src/events/SDL_pen.c

```diff
@@ -0,0 +1,114 @@
+/*
+  Simple DirectMedia Layer
+  Copyright (C) 1997-2022 Sam Lantinga <slouken@libsdl.org>
+
+  This software is provided 'as-is', without any express or implied
+  warranty.  In no event will the authors be held liable for any damages
+  arising from the use of this software.
+
+  Permission is granted to anyone to use this software for any purpose,
+  including commercial applications, and to alter it and redistribute it
+  freely, subject to the following restrictions:
+
+  1. The origin of this software must not be misrepresented; you must not
+     claim that you wrote the original software. If you use this software
+     in a product, an acknowledgment in the product documentation would be
+     appreciated but is not required.
+  2. Altered source versions must be plainly marked as such, and must not be
+     misrepresented as being the original software.
+  3. This notice may not be removed or altered from any source distribution.
+*/
+#include "../SDL_internal.h"
+
+/* Pressure-sensitive pen handling code for SDL */
+
+#include "SDL_pen_c.h"
+#include "SDL_events.h"
+#include "SDL_events_c.h"
+
+#include <stdio.h> // FIXME
```

Comment:
some left-over, I guess?

Reply:
P.S.: you needn't mention me in the commit message for such a trivial thing

Reply:
Good catch, thanks!